pkg/test/expectations/expectations.go

```diff
@@ -358,11 +359,11 @@ func ExpectMachinesCascadeDeletion(ctx context.Context, c client.Client, machine
 	}
 }
 
-func ExpectMakeMachinesReady(ctx context.Context, c client.Client, machines ...*v1alpha5.Machine) {
-	ExpectMakeMachinesReadyWithOffset(1, ctx, c, machines...)
+func ExpectMakeMachinesInitialized(ctx context.Context, c client.Client, machines ...*v1alpha5.Machine) {
```

Comment:
Frustratingly, when applying a new Node to the test api-server, there is a not-ready taint that is added by default. I couldn't figure a way to get around this, so I decided to create a new `ExpectMakeMachineInitialized` and `ExpectMakeNodeInitialized` to differentiate out the "make ready" functions from the "make initialized" functions